### DISCUSSION TITLE
普林斯顿 COS425 数据库和信息管理系统 | hackway.org

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#29] [Announcements]
# 普林斯顿 COS425 数据库和信息管理系统 | hackway.org

课程名称： Structure and Interpretation of Computer Programs

https://hackway.org/docs/cs/junior/database/cs6830

**Comment by shiququ** [comment]
你好，这个MIT数据库的链接时伯克利的链接，是不是链接放错了呀？
